templates/hound.yml

```diff
@@ -10,5 +10,6 @@ ruby:
   # config_file: .ruby-style.yml
   enabled: true
 scss:
-  # config_file: .scss-style.yml
+  enabled: false
+stylelint:
```

Comment:
We will need to add the `config_file` key in the stylelint generator. One approach is to have a comment here, the way we do with the others, and then have the generator uncomment it.

Reply:
Added